docs/feature_flags.md

```diff
@@ -194,3 +194,13 @@ won't work when you push OTLP metrics.
 
 Enables PromQL functions that are considered experimental and whose name or
 semantics could change.
+
+## Created Timestamps Zero Injection
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of created timestamps as a injection of samples with 0 value for created timestamp (when appropriate). See PromCon talk <https://youtu.be/nWf0BfQ5EEA?feature=shared> for details on the created timestamp.
```

Comment:
```suggestion
Enables ingestion of created timestamp. Created timestamps are injected as 0 valued samples when appropriate. See [PromCon talk](https://youtu.be/nWf0BfQ5EEA) for details.
```

Reply:
Don't treat my suggestions literally @ArthurSens (: `<Link>` was just semantically what would be nice, not literal structure of link in markdown (:

Reply:
My bad, for some reason I thought you were saying that this is how links are shown in the website 😅